PaymentButtons/src/main/java/com/paypal/android/paymentbuttons/PayPalCreditButton.kt

```diff
@@ -67,22 +67,36 @@ class PayPalCreditButton @JvmOverloads constructor(
 /**
  * Defines the colors available for PayPal Credit buttons.
  *
- * @see GOLD is the recommended color for PayPal Credit.
- * @see WHITE is a secondary alternative color for PayPal Credit.
- * @see DARK_BLUE is the default color if one is not provided. Deprecated - please use GOLD or WHITE.
- * @see BLACK is a secondary alternative color for PayPal Credit. Deprecated - please use GOLD or WHITE.
+ * @see DARK_BLUE is the default and recommended color for PayPal Credit.
+ * @see BLACK is a secondary alternative color for PayPal Credit.
+ * @see GOLD Deprecated. Will be removed in a future major version - please use BLACK or DARK_BLUE.
+ * @see WHITE Deprecated. Will be removed in a future major version - please use BLACK or DARK_BLUE.
```

Comment:
I only remember the part where we un-deprecate the older colors. My memory is hazy on what we were going to do with other colors.

PayPalButton had 5 colors, of which we deprecated 3. I reverted those changes, so now we have all 5 to choose from.
On the other hand, PayPalCreditButton only had 2 and they were black and blue. Reverting the changes meant removing the gold and white that were added, but since we want to be backwards compatible, we'd have to deprecate them and remove them later.


Please remind me if a decision was made to keep all 4 colors for PayPalCreditButton.

Reply:
Un-deprecated gold and white.